### DISCUSSION TITLE
b.run_solver(kind='rv_geometry', solution='rv_geom_sol') value error

### DISCUSSION THREAD

**Discussion by d11122** [phoebe-project/phoebe2#497] [Questions]
I am confusing, when I use `b = phoebe.default_binary( )`，the `b.run_solver(kind='rv_geometry', solution='rv_geom_sol')` working.
but when I use `b = phoebe.default_binary(contact_binary=True)`, the same data, the `b.run_solver(kind='rv_geometry', solution='rv_geom_sol')` not working, show`ValueError: 0 results found for twig: 'None', {'qualifier': 'rvs', 'check_visible': False, 'check_default': False, 'check_advanced': False, 'check_single': False}`.

I don't know what's going on, I hope I can get an answer, thank you very much. --daimin

**Comment by kecnry** [comment]
I just filed a bug report (#498) and will try to get a fix out sometime this week.  Thanks!

**Reply by d11122** [reply]
thanks！

**Reply by kecnry** [reply]
this was just fixed and released in the 2.3.50 bugfix release
